src/driver/main.cpp

```diff
@@ -811,7 +811,11 @@ struct main_command
 using namespace migraphx::driver; // NOLINT
 int main(int argc, const char* argv[])
 {
-    std::vector<std::string> args(argv + 1, argv + argc);
+    std::vector<std::string> cmd_args(argv, argv + argc);
+    std::string cmd_string = migraphx::to_string_range(cmd_args, " ");
```

Comment:
You can do `std::string cmd_string = std::string(argv[0]) + " " + migraphx::to_string_range(args, " ");` here to reuse the `args` vector.